packages/netlify-cms-widget-list/src/schema.js

```diff
@@ -6,4 +6,5 @@ export default {
     minimize_collapsed: { type: 'boolean' },
     label_singular: { type: 'string' },
   },
+  prohibited: ['i18n'],
```

Comment:
Don't allow a list field to have a `i18n` configuration

Reply:
This is a pretty big change from the last pull-request. 
What was the reason for not including this into the PR and what can I do to help add/implement this again? 

The reason why I ask is because I've spent this summer to implement internationalization and used the previous fork and had everything working and just introduced the entire team to my code. 😅

Reply:
Yes, going to fix this soon 😄

Reply:
Wow, that was a fast answer! 

I feel like an a**hole for asking but do you have an idea what _soon_ means?

* before this PR is merged?
* days 
* weeks
* I don't know 

The reason why I ask is because I'm trying to set expectations for my team.
Of course, if you cannot answer or do not want to answer, I totally get it. <3

Reply:
And again, point me in an direction and I'll try and help out!

Reply:
What I'll do then is to fork this PR and fix so that multicontent works with single files again and create a new PR once this one has landed. So excited!

Reply:
Before the PR is merged 😄

Reply:
The issue with lists is when using `i18n: duplicate`:
![image](https://user-images.githubusercontent.com/26760571/90549290-f5004100-e196-11ea-9fb6-308cebbcc7dc.png)

List widgets has a lot of internal state (e.g. toggle status).
To make that work in a good way for `i18n: duplicate` we would need to synchronize the internal state of widgets across editors which would require too much work at the moment.
The initial thought was to disable it completely, however it is better to at least allow `i18n: true`.